### PR TITLE
Feat/comment

### PR DESCRIPTION
**Pull request by sonishreyas**
#change logs

- Added Comment feature

### CODE REVIEW

**Review by TejasShekar — APPROVED** [review]
Clean code and good folder structure.
No suggestions.
Good to merge

File: src/features/postSlice.js [hunk 1]
```diff
@@ -158,6 +159,19 @@ export const updateLikes = createAsyncThunk(
 		}
 	}
 );
+
+export const addComment = createAsyncThunk(
+	"posts/addComment",
+	async ({ postId, updatedValue }, { rejectWithValue }) => {
+		try {
+			console.log(postId, updatedValue);
```
Comment (vidhi-thakur) [inline src/features/postSlice.js:167]: console log could be removed

File: src/features/postSlice.js [hunk 2]
```diff
@@ -279,6 +293,16 @@ const postSlice = createSlice({
 			state.explorePosts = updateArray(state.explorePosts, payload);
 			state.itemsInBookmark = updateArray(state.itemsInBookmark, payload);
 		},
+		[addComment.rejected]: (state, { payload }) => {
+			console.log(payload);
+		},
+		[addComment.fulfilled]: (state, { payload }) => {
+			console.log(payload);
```
Comment (vidhi-thakur) [inline src/features/postSlice.js:300]: could be removed if not necessary

**Review by vidhi-thakur — APPROVED** [review]
added suggestion to remove console logs otherwise looks good to merge

**Pull request merged**